GetUserInfo/GetUserInfo/Program.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace GetUserInfo
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            UserInfo UI = Questions();
+            Console.WriteLine("Do you have any pets?");
+            string response = Console.ReadLine();
+            List<string> petNames = new List<string>();
+            while (response.ToLower() == "yes")
+            {
+                Console.WriteLine("What is the name of one of your pets?");
+                string petName = Console.ReadLine();
+                petNames.Add(petName);
+                Console.WriteLine("Do you have any more pets?");
+                response = Console.ReadLine();
+            }
+            Answers(UI);
+            if (petNames.Count == 0)
+            {
+                Console.WriteLine("You do not have any animal companions.");
+            }
+            else
+            {
+                foreach (string petName in petNames)
+                {
+                    Console.WriteLine(petName + " is a loyal companion of yours.");
+                }
+            }
+        }
+
+        static UserInfo Questions()
+        {
+            UserInfo You = new UserInfo();
+            Console.WriteLine("What is your first name?");
+            You.FirstName = Console.ReadLine();
+            Console.WriteLine("What is your last name?");
+            You.LastName = Console.ReadLine();
+            You.FavoriteNumber = GetIntFromUser("What is your favorite number?");
+            return You;
+        }
+
+        static int GetIntFromUser(string prompt)
+        {
+            Console.WriteLine(prompt);
+            string response = Console.ReadLine();
+            int number = int.Parse(response);
```

Comment:
This is good and works, and I'm always of the opinion that starting with some that's working/simple is good. As an improvement, could this be written so that it won't throw an exception if the user doesn't respond with an integer? Probably `int.TryParse` instead of `int.Parse`?